quipucords/api/deployments_report/tests_deployments_report.py

```diff
@@ -323,3 +326,44 @@ def test_csv_renderer_only_name(self):
         # pylint: disable=line-too-long
         expected = 'Report ID,Report Type,Report Version\r\n1,deployments,%s\r\n\r\n\r\nSystem Fingerprints:\r\narchitecture,bios_uuid,cpu_core_count,cpu_count,cpu_socket_count,detection-network,detection-satellite,detection-vcenter,entitlements,infrastructure_type,ip_addresses,is_redhat,mac_addresses,name,os_name,os_release,os_version,redhat_certs,redhat_package_count,sources,subscription_manager_id,system_creation_date,system_last_checkin_date,virtualized_type,vm_cluster,vm_datacenter,vm_dns_name,vm_host,vm_host_socket_count,vm_state,vm_uuid\r\n,,,,,,,,,,,,,1.2.3.4,,,,,,,,,,,,,,,,,\r\n,,,,,,,,,,,,,1.2.3.4,,,,,,,,,,,,,,,,,\r\n,,,,,,,,,,,,,1.2.3.4,,,,,,,,,,,,,,,,,\r\n\r\n' % self.report_version  # noqa
         self.assertEqual(csv_result, expected)
+
+    ##############################################################
+    # Test Json Gzip Render
+    ##############################################################
+    def test_json_gzip_renderer(self):
```

Comment:
Let's add tests for report ID not found. We should have these for all content types we return (should means ideally)